indico/modules/events/surveys/controllers/management/questionnaire.py

```diff
@@ -46,6 +52,82 @@ def _process(self):
                                               field_types=field_types, preview_form=preview_form)
 
 
+class RHExportSurveyQuestionnaire(RHManageSurveyBase):
+    """Export the questionnaire to JSON format"""
+
+    def _process(self):
+        sections = [section.to_dict() for section in self.survey.sections]
+        response = jsonify(version=1, sections=sections)
+        response.headers['Content-Disposition'] = 'attachment; filename="survey.json"'
+        return response
+
+
+class RHImportSurveyQuestionnaire(RHManageSurveyBase):
+    """Import a questionnaire in JSON format"""
+
+    def _import_data(self, data):
+        if data['version'] != 1:
+            raise ValueError("Unsupported document format")
+
+        # Remove the default section added to new surveys
+        if len(self.survey.sections) == 1:
```

Comment:
Actually, I think you can simplify this whole check:

    if len(self.survey.items) == 1 and isinstance(self.survey.items[0], SurveySection)